mlflow/pyfunc/model.py

```diff
@@ -925,7 +930,8 @@ def predict_stream(
                 "`predict_stream` method on your model to generate streaming predictions"
             )
 
-        def create_text_delta(self, delta: str, item_id: str) -> dict[str, Any]:
+        @staticmethod
+        def create_text_delta(delta: str, item_id: str) -> dict[str, Any]:
```

Comment:
Not a blocker, but do we really need so many helper functions? Can we align them with one function, they're just constructing a dictionary